### ISSUE TITLE
New message window doesn't show which account you are posting from

### ISSUE THREAD

**Issue by lzaldivarkt** [NicolasConstant/sengi#477]
It's hard to distinguish from which account you are posting from from the message window.

**Comment by NicolasConstant** [comment]
Every action you do with Sengi is done with the currently selected account on the left (the non transparent/dark ones). But you're not the first one referring to this issue when posting, an option to enable a small reminder near the posting area might help a bit on this part.

**Comment by lzaldivarkt** [comment]
Yeah, that's fair. If you have multiple accounts with similar avatars it might be tricky to identify which one is it.
